include/maidsafe/routing/types.h

```diff
@@ -85,6 +85,9 @@ using HandleGetReturn =
     boost::expected<boost::variant<std::vector<DestinationAddress>, std::vector<byte>>,
                     maidsafe_error>;
 using HandlePutPostReturn = boost::expected<std::vector<DestinationAddress>, maidsafe_error>;
+using HandlePostReturn =
```

Comment:
agreed, it's just I don't want to mess up with stuff when trying to replace HandlePutPostReturn with HandlePutReturn across project in this pull request.
And also such change need to be reviewed and discussed before being carried out